README.md

```diff
@@ -55,6 +63,22 @@ Please find below the requirements to complete this course:
 
 If you need help in any of these topics, [a tutorial is available on the pre-class](classes/00pre-class/README.md):
 
+# Live/Online Applications
+
+If you intend to apply for the Live/Online classes, you'll need to fill the form available [here](https://www.devopsacademy.com.au), and as part of your application, you'll need to submit a PR to this repository under the applications folder following the instructions below. If there is no form available on the link provided, there is no scheduled classes in place. Reach out to any of us through our Slack channel to get more information on next Live/Online classes.
```

Comment:
can you make some space after the first paragraph to be more clear about when no registration is open

Reply:
fixed